dev/com.ibm.ws.jdbc/src/com/ibm/websphere/ce/cm/ConnectionWaitTimeoutException.java

```diff
@@ -21,11 +21,127 @@
     private static final long serialVersionUID = 5958695928250441720L;
 
     /**
-     * Make a new ConnectionWaitTimeoutException.
-     * 
-     * @param message the exception message.
+     * Constructs a <code>ConnectionWaitTimeoutException</code> object.
+     * The <code>reason</code>, <code>SQLState</code> are initialized
+     * to <code>null</code> and the vendor code is initialized to 0.
+     *
+     * The <code>cause</code> is not initialized, and may subsequently be
+     * initialized by a call to the
+     * {@link Throwable#initCause(java.lang.Throwable)} method.
```

Comment:
Done.  Since I pulled the constructor methods and javadoc straight from the SQLTransientConnectionException parent class, should I remove the Throwable#initCause() text from the javadoc there as well?

Reply:
`SQLTransientConnectionException` comes from the JDK, so we can't modify that class or its doc

Reply:
D'OH!  Good catch... :-)